webrender/src/renderer.rs

```diff
@@ -194,37 +194,52 @@ pub enum BlendMode {
 
 /// The device-specific representation of the cache texture in gpu_cache.rs
 struct CacheTexture {
-    id: TextureId,
+    current_id: TextureId,
+    next_id: TextureId,
 }
 
 impl CacheTexture {
     fn new(device: &mut Device) -> CacheTexture {
-        let id = device.create_texture_ids(1, TextureTarget::Default)[0];
+        let ids = device.create_texture_ids(2, TextureTarget::Default);
 
         CacheTexture {
-            id: id,
+            current_id: ids[0],
+            next_id: ids[1],
```

Comment:
given the rarity (and limitedness) of this resize operation, I wouldn't bother keeping a separate texture ID and would just create one on the spot (temporarily)